test/fixtures/google-gax-packaging-test-app/src/index.js

```diff
@@ -131,11 +134,29 @@ async function testPagedExpand(client) {
     content: words.join(' '),
     pageSize: 2,
   };
+  const timer = setTimeout(() => {throw Error('End-to-end testPagedExpand method fails with timeout')}, 12000);
   const [response] = await client.pagedExpand(request);
+  clearTimeout(timer);
   const result = response.map(r => r.content);
   assert.deepStrictEqual(words, result);
 }
 
+async function testPagedExpandAsync(client) {
+  const words = ['nobody', 'ever', 'reads', 'test', 'input'];
+  const request = {
+    content: words.join(' '),
+    pageSize: 2,
+  };
+  const response = [];
+  const iterable = client.pagedExpandAsync(request);
+  const timer = setTimeout(() => {throw Error('End-to-end testPagedExpandAsync method fails with timeout')}, 12000);
```

Comment:
`throw new Error`. Also, as a nit, can you format it into several lines? This line is too long. Unfortunately, we don't run `eslint` / `prettier` for this folder.